### ISSUE TITLE
[FEATURE] Show image path

### ISSUE THREAD

**Issue by MiroHofo** [ImageOptim/ImageOptim#205]
Add option to show file path (column) in the list of images, not only file name.

**Comment by pornel** [comment]
If you hover your mouse over loupe (show in finder) icon, it'll show the full path. Does this work for you?

**Comment by MiroHofo** [comment]
Not really, it kind of works but it is very slow way of doing this and won't be visible of screenshots for multiple images.

**Comment by pornel** [comment]
Ah, so I guess you're trying to use screenshots as a solution to #206 ?

**Comment by MiroHofo** [comment]
Well it was one of the use (but not working with 100+ images :)). Motivation behind this is, that I am using ImageOptim for whole projects which has hundreds of images spread across different directories and often with same name (especially with uploaded photos with same name but different sizes in separate folders). In this case is loupe very slow to use to distinguish what image is that and results cannot be sorted according to file path. There is definitelly use case to have file path displayed for me and from UX perspective, there is value as well ;)
